sdk/eventhub/event-hubs/README.md

```diff
@@ -370,14 +370,14 @@ async function main() {
   };
 
   const subscription = client.subscribe(partitionIds[0], {
-    processEvents: (events, context) => {
-      // event processing code goes here
+      processEvents: (events, context) => {
```

Comment:
```suggestion
      processEvents: async (events, context) => {
```

??

Reply:
It's not actually an error (since it's JS).

Reply:
The one I fixed is only an error because it's also calling a function with `await`

Reply:
ok